docs/scos/dev/tutorials-and-howtos/howtos/howto-set-up-multiple-stores.md

```diff
@@ -49,7 +49,7 @@ Multi-store setup 3: Each store has a dedicated database, search engine, and key
 
 This setup is most suitable for projects with the following requirements:
 
-* Completely different business requirements per store, like business logic and features.
+* Completely different business requirements per store, such as business logic and features.
```

Comment:
We use `such as` only for inclusive examples, meaning you should enlist all the items related to the described category. When `like` is used, it is ok to list some of the items related to the category.